exportjson.lua

```diff
@@ -172,7 +172,10 @@ do
 				end
 			end
 		end
-		local _, activityID = C_LFGList.GetActiveEntryInfo()
+		local activityEntryInfo = C_LFGList.GetActiveEntryInfo()
+
+		local activityID = activityEntryInfo['activityID']
```

Comment:
could just do activityEntryInfo.activityID here right? I think we index into tables like that more consistently in this code?